lib/wibox/widget/slider.lua

```diff
@@ -303,6 +325,14 @@ function slider:set_value(value)
     end
 end
 
+--- Returns `true` while the user is dragging the handle.
+--
+-- @method is_dragging
```

Comment:
sorry for missing this earlier, this should be `@property` (there is magic to make it work)

Reply:
Do you mean "transform this into a property by adding `get_is_dragging` and `set_is_dragging`?"
Because there is currently no magic involved. Indexing `.is_dragging` returns a function, not a boolean.